src/test/java/com/github/dzieciou/testing/curl/CurlLoggingInterceptorTest.java

```diff
@@ -109,6 +109,148 @@ public void shouldLogStacktraceWhenEnabled() {
         .and(containsString(("java.lang.Thread.getStackTrace"))));
   }
 
+  @Test
```

Comment:
Instead of creating multiple methods that differ only in logging level in `options` and expected logging level in logs, I suggest to turn those tests into one parametrized test using, for instance, TestNG `DataProvider`, see example [here](https://testng.org/doc/documentation-main.html#parameters-dataproviders).